<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelWelcome">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>471</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Quicksand;
font-style: normal;
font-weight: 400;
font-size: 18px;
line-height: 23px;
color: #2B2D2F;
display: flex;
align-items: center;
text-align: center;</string>
    </property>
    <property name="text">
     <string>Shot New Zealand! Together we can save lives.  </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAccessNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>461</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Quicksand;
font-style: normal;
font-weight: 400;
font-size: 18px;
line-height: 23px;
color: #2B2D2F;
display: flex;
align-items: center;
text-align: center;</string>
    </property>
    <property name="text">
     <string>Access Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogin">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>176</y>
      <width>520</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Quicksand;
font-style: normal;
font-weight: 400;
font-size: 23px;
line-height: 23px;
color: #2B2D2F;
display: flex;
align-items: center;
text-align: center;colour: black;</string>
    </property>
    <property name="text">
     <string> Login / Sign-Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-5</y>
      <width>1101</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>641</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
      <pointsize>44</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Quicksand;
font-style: normal;
font-weight: 400;
font-size: 44pt;
line-height: 23px;
color: #2B2D2F;
display: flex;
align-items: center;
text-align: center;</string>
    </property>
    <property name="text">
     <string>Unite Against COVID-19</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNHI">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>461</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Quicksand;
font-style: normal;
font-weight: 400;
font-size: 18px;
line-height: 23px;
color: #2B2D2F;
display: flex;
align-items: center;
text-align: center;colour: black;</string>
    </property>
    <property name="text">
     <string>NHI Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEmail">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>217</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Quicksand;
font-style: normal;
font-weight: 400;
font-size: 18px;
line-height: 23px;
color: #2B2D2F;
display: flex;
align-items: center;
text-align: center;colour: black;</string>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelError">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>565</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 542px;
height: 77px;
left: 54px;
top: 486px;
background:#FFFFFF;
box-sizing: border-box;
border-radius: 20px;
font-family: Quicksand;
font-style: normal;
font-weight: 600;
font-size: 14px;
line-height: 33px;
/* or 143% */

display: flex;
align-items: center;
color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgHome">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>571</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource.qrc">:/res/images/dp.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="editEmail">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>255</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 542px;
height: 77px;
left: 54px;
top: 486px;
background:#FAFCFF;
border: 2.3px solid #979797;
box-sizing: border-box;
border-radius: 20px;
font-family: Quicksand;
font-style: normal;
font-weight: 600;
font-size: 20px;
line-height: 33px;
/* or 143% */

display: flex;
align-items: center;
color: gray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editNHI">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 542px;
height: 77px;
left: 54px;
top: 486px;
background:#FAFCFF;
border: 2.3px solid #979797;
box-sizing: border-box;
border-radius: 20px;
font-family: Quicksand;
font-style: normal;
font-weight: 600;
font-size: 20px;
line-height: 33px;
/* or 143% */

display: flex;
align-items: center;
color: gray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editAccessNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 542px;
height: 77px;
left: 54px;
top: 486px;
background:#FAFCFF;
border: 2.3px solid #979797;
box-sizing: border-box;
border-radius: 20px;
font-family: Quicksand;
font-style: normal;
font-weight: 600;
font-size: 20px;
line-height: 33px;
/* or 143% */

display: flex;
align-items: center;
color: gray;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLogin">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 542px;
height: 77px;
left: 54px;
top: 801px;

background: rgba(252, 219, 100, 0.55);
border: 2.3px solid #FFB61D;
box-sizing: border-box;
border-radius: 20px;
font-family: Quicksand;
font-style: normal;
font-weight: 600;
font-size: 23px;
line-height: 23px;
color: #2B2D2F;
display: flex;
align-items: center;
text-align: center;</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <zorder>labelBackground</zorder>
   <zorder>labelWelcome</zorder>
   <zorder>labelAccessNumber</zorder>
   <zorder>labelLogin</zorder>
   <zorder>labelNHI</zorder>
   <zorder>labelEmail</zorder>
   <zorder>labelError</zorder>
   <zorder>imgHome</zorder>
   <zorder>labelTitle</zorder>
   <zorder>editEmail</zorder>
   <zorder>editNHI</zorder>
   <zorder>editAccessNumber</zorder>
   <zorder>pbLogin</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
